Lesson4/homework4.py

```diff
@@ -0,0 +1,70 @@
+# Task 1
+print('Task 1: \"Removing letters\"')
+fruits = 'apple mango banana'
+my_str = input('Enter string: ')
+letters = input('Enter letter to delete: ')
+result1 = my_str.replace((letters), '')
```

Comment:
You don't need to use () for letters in replace. Correct variant
result1 = my_str.replace(letters, '')